CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# matrix-sdk-crypto-wasm unreleased
+
+-   Functions/methods that take a JavaScript `Array` as argument now delete the
+    array's items once their return, for example `Device::requestVerification`,
```

Comment:
AIUI, it's not that the entries are deleted from the array, but rather that the individual items are themselves invalidated so that they can no longer be used?

Also: I think it happens as soon as the method is called, rather than once it returns? In other words, I can't do this:

```javascript
    const user = new UserId("@user:domain");
    await Promise.all([
         machine.query_keys_for_users([user]),
         // `user` is used below before `query_keys_for_users` returns, but AIUI this will fail.
         machine.update_tracked_users([user]),
     ]);
```

```suggestion
-   Functions/methods that take a JavaScript `Array` as argument now invalidate the
    items within that array so that they cannot be re-used. Affected methods include `Device::requestVerification`,
```